test/runtests.jl

```diff
@@ -12,12 +11,14 @@ solvers = [
            ODE.ode4,
            ODE.ode4ms,
            ODE.ode5ms,
+           ODE.ode_am,
```

Comment:
Yes, thanks for the catch. `ode_am` defaults to `ode4am`, but it is best to be more precise.